SYSTEMS.md

```diff
@@ -137,16 +137,3 @@ reframe -c examples/sombrero -r --performance-report --system tesseract:compute-
 ```
 
 where `<ACCOUNT>` is the project you want to charge.
-
-## Tursa
-
-### Queue options
-
-When submitting jobs to compute nodes, you need to specify the job queue, with the `--account` option to the scheduler.
-To do this, when you run a benchmark you can use the `-J`/`--job-option` flag to `reframe` to specify the account, for example:
-
-```
-reframe -c examples/sombrero -r --performance-report --system tursa:compute-node -J'--accout=<ACCOUNT>'
-```
-
-where `<ACCOUNT>` is the project you want to charge.
```

Comment:
How confident are you that this will always work? Would it be better to keep this section but rephrase it as "We will try to automatically detect your account, but in case this fails..."

Reply:
This is what they do in [CSCS Piz Daint reframe benchmarks](https://github.com/reframe-hpc/cscs-reframe-tests/blob/10ac6b5ac0a08c1909fee742fccd0b87ef4610cd/config/systems/daint.py#L49), where I took inspiration for this.  Sounds like they're highly confident this might work?